ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-security.j2

```diff
@@ -0,0 +1,61 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - ip security #}
+{% if ip_security is arista.avd.defined %}
+
+### IP Security
+{%     if ip_security.ike_policie is arista.avd.defined %}
+
+##### IKE policies
```

Comment:
Something looks weird with the number of #.

Reply:
yup - my bad !